client/components/price/price-filter.js

```diff
@@ -52,12 +52,12 @@ function priceFilter($translate) {
       }
       if (frequency === 'yearly') {
         return `<span class="bold">${$translate.instant('common_price_ht_label', { price: price.withoutTax.text })}`
-                          + `<small>${$translate.instant('common_price_label_yearly')}</small></span> `
-                          + `<span class="italic small"> (${$translate.instant('common_price_ttc_label', { price: price.withTax.text })}`
-                          + `<small>${$translate.instant('common_price_label_yearly')}</small>)</span>`;
+          + `<small>${$translate.instant('common_price_label_yearly')}</small></span> `
```

Comment:
Not blocking but, why using string concatenation for multilines text with template string (that have multilines support)?